net.devolutions.RDM.yml

```diff
@@ -0,0 +1,38 @@
+app-id: net.devolutions.RDM
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: remotedesktopmanager
+tags: [proprietary]
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --filesystem=host
+   
+modules:
+  - name: rdm
+    buildsystem: simple
+    build-commands:
+      - ar x RemoteDesktopManager.deb
+      - tar xf data.tar.xz
+      - mv usr/* /app/
+      - rm -r control.tar.xz data.tar.xz debian-binary
+      - install -D remotedesktopmanager /app/bin/remotedesktopmanager
+      - install -Dm644 net.devolutions.RDM.desktop ${FLATPAK_DEST}/share/applications/net.devolutions.RDM.desktop
+      - install -Dm644 net.devolutions.RDM.png ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/net.devolutions.RDM.png
+    sources:
+      - type: file
+        url: https://cdn.devolutions.net/download/Linux/RDM/2022.1.2.2/RemoteDesktopManager_2022.1.2.2_amd64.deb
+        dest-filename: RemoteDesktopManager.deb
+        sha256: 4c2ae071d09e8aa38ad8b48e28733e238b8c7746c897a1ff98d14e52fb1b44cd
+        only-arches:
+          - x86_64
+      - type: file
+        path: remotedesktopmanager
+      - type: file
+        path: net.devolutions.RDM.desktop
+      - type: file
+        path: net.devolutions.RDM.png
```

Comment:
this icon is also in the .deb. No need to have it here.

It is in ./usr/share/icons/remotedesktopmanager.png and should be moved to the right place.